jbpm-system-test/jbpm-springboot-mssql-test/pom.xml

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jbpm</groupId>
+    <artifactId>jbpm-system-test</artifactId>
+    <version>7.61.0.Final</version>
+  </parent>
+  
+  <artifactId>jbpm-springboot-mssql-test</artifactId>
+
+  <name>jBPM Springboot MSSQL Test</name>
+  <description>jBPM Springboot with MSSQL</description>
+
+  <properties>
+    <start-class>org.kie.server.springboot.samples.JBPMApplication</start-class>
+    <version.kie-server-spring-boot-starter>7.59.1-SNAPSHOT</version.kie-server-spring-boot-starter>
+    <jbpm.branch>7.59.x</jbpm.branch>
```

Comment:
Same here. Why not using the same project version? is it not aligned with the repos in jbpm? wdyt?

Reply:
Right, it is not always possible to download the ddl from the same branch as Final image (they are removed periodically)